src/main/java/ua/ithillet/lesson6/WorkWithListCollection.java

```diff
@@ -0,0 +1,69 @@
+package ua.ithillet.lesson6;
+
+import java.util.*;
+
+public class WorkWithListCollection {
+
+    public static int countOccurrence(List<String> list, String searchItem) {
+        int count = 0;
+        for (String item : list) {
+            if (item.equals(searchItem)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(ArrayList<Integer> list) {
```

Comment:
```suggestion
    public static List<Integer> toList(Integer[] list) {
```
такой має бути інтерфейс